content/concepts/consensus-network/invariant-checking.md

```diff
@@ -130,6 +130,25 @@ The XRP Ledger checks all the following invariants on each transaction:
     - A new account root must have the right starting [sequence](basic-data-types.html#account-sequence).
     - A transaction must not create more than one new [account](accounts.html).
 
+### ValidNFTokenPage
+
+- **Invariant Condition(s):**
+    - The number of minted or burned NFTs can only be changed by `NFTokenMint` or `NFTokenBurn` transactions.
+    - A successful NFTokenMint transaction must increase the number of NFTs.
+    - A failed NFTokenMint transaction must not change the number of minted NFTs.
+    - A NFTokenMint transaction cannot change the number of burned NFTs.
+    - A successful NFTokenBurn transaction must increase the number of burned NFTs.
+    - A failed NFTokenBurn transaction must not change the number of burned NFTs.
+    - A NFTokenBurn transaction cannot change the number of minted NFTs.
+    
+### NFTokenCountTracking
+- **Invariant Condition(s):**
```

Comment:
Leave a blank line before and after bulleted lists to make sure they parse correctly.

```suggestion
### NFTokenCountTracking

- **Invariant Condition(s):**
```